portfolio/src/main/webapp/script.js

```diff
@@ -12,17 +12,47 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+window.onload = function() {
+  setTimeout(toggleContent, 3500);
+}
+
 /**
- * Adds a random greeting to the page.
+ * Generate a random fact about myself
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function randomFact() {
+  const facts = [
+    'I can play the guitar!', 
+    'I broke my thumb before...',
+    'My hair turned brown from swimming (chlorine).',
+    'I love singing.',
+    'I am a Hong Kong PR.'
+  ];
+  var fact = facts[Math.floor(Math.random() * facts.length)];
```

Comment:
this variable doesnt seem to be modified anymore, can we use const on it?

Reply:
You're right, wouldn't be changed once assigned. Edited it to use const!